central/clusters/deployer_test.go

```diff
@@ -289,6 +289,16 @@ func testMetaValueGenerationWithImageFlavor(s *deployerTestSuite, flavor default
 	}
 }
 
+func testImageFlavorChartRepoSettings(s *deployerTestSuite, flavor defaults.ImageFlavor) {
+	cluster := makeTestCluster(flavor.MainImage(), flavor.CollectorFullImage())
+
+	fields, err := FieldsFromClusterAndRenderOpts(cluster, &flavor, RenderOptions{})
+	s.NoError(err)
+	s.Equal(flavor.ChartRepo.URL, fields.ChartRepo.URL, "ChartRepo URL does not match")
+	s.Equal(flavor.ChartRepo.IconURL, fields.ChartRepo.IconURL, "ChartRepo IconURL does not match")
```

Comment:
I'd also assert that `fields.ChartRepo.*` fields aren't empty here. I double-checked and all flavors are setting both the chart URL and the IconURL. However, if in the future someone adds a new flavor and forgets to specify an `IconURL` this test will pass, even though there shouldn't be an empty string as `IconURL`.

Reply:
Good idea, done!